packages/gallery/__tests__/playwright/playwright.spec.js

```diff
@@ -0,0 +1,75 @@
+const { test, expect } = require('@playwright/test');
+import defaults from '../../src/lib/defaults';
+import AxeBuilder from '@axe-core/playwright';
+
+test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+});
+
+test.describe('Gallery > Keyboard Navigation', { tag: '@all' }, () => {
+    test('Should navigate to the next item when the right arrow key is pressed', async ({ page }) => {
+        const list = await page.locator('[data-gallery-list]');
+        const items = await page.locator('[data-gallery-item]').all();
+        await list.focus();
+        await expect(items[0]).toHaveClass(`gallery__item ${defaults.className.active}`);
+        await page.keyboard.press('ArrowRight');
+        await expect(items[1]).toHaveClass(`gallery__item ${defaults.className.active}`);
+        await page.keyboard.press('ArrowRight');
+        await expect(items[2]).toHaveClass(`gallery__item ${defaults.className.active}`);
+    });
+
+    test('Should navigate to the previous item when the left arrow key is pressed', async ({ page }) => {
+        const list = await page.locator('[data-gallery-list]');
+        const items = await page.locator('[data-gallery-item]').all();
+        await list.focus();
+        await expect(items[0]).toHaveClass(`gallery__item ${defaults.className.active}`);
+        await page.keyboard.press('ArrowLeft');
+        await expect(items[items.length - 1]).toHaveClass(`gallery__item ${defaults.className.active}`);
+        await page.keyboard.press('ArrowLeft');
+        await expect(items[items.length - 2]).toHaveClass(`gallery__item ${defaults.className.active}`);
+    });
+});
+
+test.describe('Gallery > Full screen mode support', { tag: '@all' }, () => {
```

Comment:
I think this test set is something different now!  It should maybe be 'Gallery > URL Update' or something like that